Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -25,65 +25,73 @@
             "mandatory": true,
             "display_name": "Filters And Transformers"
         },
-        "qualys": {
+        "GCP-Enrichment-Remediation": {
             "mandatory": false,
-            "display_name": "Qualys"
+            "display_name": "GCP Enrichment and Remediation"
         },
         "Rapid7_Nexpose": {
             "mandatory": false,
             "display_name": "Rapid7 InsightVM"
         },
-        "GCP-Enrichment-Remediation": {
-            "mandatory": false,
-            "display_name": "GCP Enrichment and Remediation"
+        "PrismaCloud": {
+            "mandatory": true,
+            "display_name": "Prisma Cloud by Palo Alto Networks"
         },
-        "AWS-Enrichment-Remediation": {
-            "mandatory": false,
-            "display_name": "AWS Enrichment and Remediation"
+        "ServiceNow": {
+            "mandatory": true,
+            "display_name": "ServiceNow"
         },
-        "CommonScripts": {
+        "Active_Directory_Query": {
             "mandatory": true,
```

Comment:
This and several others have been changed to TRUE which can be a breaking change, please switch them back to False unless they were previously TRUE

Reply:
Reverted back the changes and added only required changes.